src/extension.ts

```diff
@@ -160,8 +161,8 @@ async function isQuarkusProject(): Promise<boolean> {
   for (const ws of workspace.workspaceFolders) {
     const buildFileUris = await getFilePathsFromWorkspace(ws, "**/{pom.xml,build.gradle}");
     for (const uri of buildFileUris) {
-      const doc = await workspace.openTextDocument(uri);
-      if (doc.getText().search("io.quarkus") > 0) {
+      const doc = readFileSync(uri.fsPath);
```

Comment:
It might be better to read the file asynchronously. I think you you use the function returned by `promisify(readFile)` that should work?